scripts/protect_readmes.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+# Small script to verify the READMEs that get pulled in automatically don't get changed in a pull request
+
+import argparse
+import subprocess
+import os
+
+
+def file_has_changed(filename, branch):
+    """
+    Verify that the file has not been changed.
+    """
+    if not os.path.isfile(filename):
+        print(f"Error: File not found: {filename}")
+        exit(1)
+
+    try:
+        output = subprocess.check_output(['git', 'diff', '--exit-code', f'{branch}..main', '--', filename])
+        # If the output is empty, it means the file has not been changed
+        return bool(output.strip())
+    except subprocess.CalledProcessError:
+        # If git diff returns a non-zero exit code, it means the file has been changed
+        return True
+
+
+def main():
+    """
+    Main function to parse command-line arguments and execute the script.
+    """
+    parser = argparse.ArgumentParser(description='Parse a Markdown file and replace relative links with absolute links.')
+    parser.add_argument('input_file', help='Path to the input file containing source and target paths')
+    parser.add_argument('branch', help='Name of the base branch to compare to `main`')
+
+    args = parser.parse_args()
+
+    if not os.path.isfile(args.input_file):
+        print(f"Error: File not found: {args.input_file}")
+        exit(1)
+
+    with open(args.input_file, 'r') as input_file:
+        for line in input_file:
+            # Skip comments
+            if line.strip().startswith("#"):
+                continue
+
+            source, targetpath = line.strip().split(':')
+
+            if source and targetpath:
+                if file_has_changed(targetpath, args.branch):
+                    url = source.replace('/main','/blob/main')
+                    print(f"⛔ Error: The file {targetpath} cannot be manually changed.\n"
+                          f"   It is pulled in periodically from 'http://github.com/osbuild/{url}'.")
+                    exit(1)
```

Comment:
Maybe let's not exit immediately, and let the script check all files before exiting?